CMakeLists.txt

```diff
@@ -89,6 +90,7 @@ IF (WITH_DTRACE)
 ENDIF ()
 
 ADD_LIBRARY(quicly ${QUICLY_LIBRARY_FILES})
+TARGET_LINK_LIBRARIES(quicly LINK_PUBLIC m)
```

Comment:
I'm much more familiar with newer CMake versions where interface dependencies are better supported. Since your minimum required version is 2.8.11, I found this solution to define them without having to set a policy. Public/interface shouldn't make much of a difference for a library.